iot-lambda-pub-receiver-cdk/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lambda-iot-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "lambda-iot-cdk": "bin/lambda-iot-cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "aws-cdk": "2.165.0",
+    "ts-node": "^10.9.2",
+    "typescript": "~5.6.3"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-iam": "^1.203.0",
+    "@aws-cdk/aws-iot-actions-alpha": "^2.165.0-alpha.0",
+    "@aws-cdk/aws-iot-alpha": "^2.165.0-alpha.0",
+    "@aws-cdk/aws-lambda": "^1.203.0",
+    "aws-cdk-lib": "2.165.0",
+    "constructs": "^10.0.0",
+    "source-map-support": "^0.5.21"
```

Comment:
pls update using CDK v2 libraries, as CDK v1 has reached end of support 2023-06-01